101-devtestlab-create-vm-builtin-user/metadata.json

```diff
@@ -0,0 +1,7 @@
+{
+  "itemDisplayName": "Create a new VM in a DevTestLab instance",
```

Comment:
These values look the same for the first template above, needs to be different for this template